src/utils/update/mapper.utils.ts

```diff
@@ -10,7 +10,7 @@ type DateOfBirthTypes = {
 
 export const mapDateOfBirth: DateOfBirthTypes = (dateOfBirth) => {
   return {
-    day: dateOfBirth?.day,
+    day: dateOfBirth?.day ?? 1,
```

Comment:
Should this be "01"? Just to get the typing right (string) and be consistent int padding?

Reply:
@AndreKramer7654 Added.